packages/rancher-prom-to-teams/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,8 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,3 +1,5 @@
++annotations:
++  catalog.cattle.io/hidden: "true"
+ apiVersion: v1
+ appVersion: "3.2.1"
+ description: A Helm chart for Prom2Teams
```

Comment:
nit: give credit to the original upstream in the description by adding a link to it